<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>senales cerebrales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Panel Canal</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rango de muestras</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="start"/>
      </item>
      <item>
       <widget class="QSpinBox" name="final_2">
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="boton_rango">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fin</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Canal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="num_canal"/>
      </item>
      <item>
       <widget class="QPushButton" name="boton_mostrar">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Panel filtrado</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>148</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="pond">
        <item>
         <property name="text">
          <string>One</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single Level Noise (sln)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multiple Level Noise (mln)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="umb">
        <item>
         <property name="text">
          <string>Hard treshold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Soft treshold</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lamdita">
        <item>
         <property name="text">
          <string>Universal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimax</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SURE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_filtrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ponderaci칩n:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Umbral:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Lambda:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Panel operaciones basicas</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Escalamiento en amplitud</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Desplazamiento en el tiempo</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="boton_disminuir">
        <property name="text">
         <string>Disminuir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_aumentar">
        <property name="text">
         <string>Aumentar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="boton_atras">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_adelante">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="boton_guardar">
       <property name="text">
        <string>Guardar se침al</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_cargar">
       <property name="text">
        <string>Cargar se침al</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Rango de frecuencias para an치lisis frecuencial (Hz)</string>
    </property>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hasta</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="inicio_freq">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="final_freq">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Desde</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>70</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Analisis Welch</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametros:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>frecuencia de muestreo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de ventana:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tama침o de ventana:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tipo_ventana_welch">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>146</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>boxcar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>triang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blackman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hamming</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hann</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bartlett</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>flattop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>parzen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bohman</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="mostrar_welch">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>146</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Welch</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="freq_muestreo_welch">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="tamano_ventana_welch">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Solapamiento:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="solapamiento_welch">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Analisis Multitaper</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametros:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>frecuencia de muestreo:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_multitaper">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>146</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Multitaper</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Tapers</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ancho de banda (Hz)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="W_multitaper">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duraci칩n de datos(s)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="T_multitaper">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de tapers</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numero_tapers_multitaper">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string>fpass</string>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>F min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>F max</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="f_min_multitaper">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="f_max_multitaper">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Reshape</string>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1er Parametro</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="first_parameter_multitaper">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="freq_muestreo_multitaper">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>30</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Analisis Wavelet Continuo</string>
    </property>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parametros:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>frecuencia de muestreo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de muestras</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mostrar_wavelet">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>146</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Wavelet_Continuo</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="freq_muestreo_wavelet">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="muestras_wavelet">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Banda de frecuencia a analizar</string>
     </property>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>F min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>F max</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="f_min_wavelet">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="f_max_wavelet">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
